src/main/resources/crafter/studio/studio-config.yaml

```diff
@@ -105,6 +105,8 @@ studio.repo.git.cli.process.destroy.waitForTimeout: 60
 studio.blob.config.module: studio
 # Location of the blob store configuration file in the site
 studio.blob.config.path: blob-stores-config.xml
+# Location of the default blob store configuration file if the site does not have one on site create
+studio.blob.default.config.path: classpath:crafter/studio/serverless/default-blob-stores-config.xml
```

Comment:
Can I override this file by putting it under `CRAFTER_INSTALL/shared/classes/crafter/studio/serverless/default-blob-stores-config.xml`? Or I have to change the property to the exact path in the file system? (like `file:/opt/crafter/blabla...`)

Reply:
You have to change the property. You can use a classpath location though. Something like:
```
studio.blob.default.config.path: classpath:crafter/studio/serverless/override-blob-stores-config.xml
```